astropy/cosmology/units.py

```diff
@@ -80,7 +82,7 @@ def dimensionless_redshift() -> Equivalency:
 def redshift_distance(
     cosmology: Cosmology | str | None = None,
     kind: Literal["comoving", "lookback", "luminosity"] = "comoving",
-    **atzkw: Any,
+    **atzkw: Unpack[ZAtValueKWArgs],
```

Comment:
With `**atzkw` it is clearly better to use `ZAtValueKWArgs` in the annotation instead of the completely obscure `Any`.